src/main/java/com/johncnstn/auth/security/DomainUserDetailsService.java

```diff
@@ -20,11 +20,11 @@
     @Override
     @Transactional(readOnly = true)
     @SuppressWarnings("")
-    public DomainUserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+    public DomainUserDetails loadUserByUsername(String username) {
         return loadUserByEmail(username);
     }
 
-    private DomainUserDetails loadUserByEmail(String email) throws UsernameNotFoundException {
+    private DomainUserDetails loadUserByEmail(String email) {
```

Comment:
This throws UsernameNotFoundException have to be returned, thought.

Reply:
Left it as it is, as per offline discussion.